syntax = "proto3";

service NotificationService {
  rpc TriggerNotification (NotificationRequest) returns (NotificationResponse);
}

message NotificationRequest {
  string message = 1;
}

message NotificationResponse {
  string result = 1;
}
